<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Edit</class>
 <widget class="QDialog" name="Edit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit a scientist</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayoutName">
    <item>
     <widget class="QLabel" name="labelName">
      <property name="text">
       <string>Name:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditPeopleName"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>20</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="HorizontalLayoutGender">
    <item>
     <widget class="QLabel" name="labelGender">
      <property name="text">
       <string>Gender:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBoxGender">
      <item>
       <property name="text">
        <string>Female</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Male</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacerGender">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayoutBirth">
    <item>
     <widget class="QLabel" name="labelBirth">
      <property name="text">
       <string>Birth:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditPeopleBirth"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>80</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayoutDeath">
    <item>
     <widget class="QLabel" name="labelDeath">
      <property name="text">
       <string>Death:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditPeopleDeath"/>
    </item>
   </layout>
  </widget>
  <widget class="QTextEdit" name="textEditPeopleInfo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>451</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelInfo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Info:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSubmit">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>280</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit changes</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonCancel">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelEditPeopleError">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
